modules/distr-tracing-tempo-config-receiver-tls-for-tempomonolithic.adoc

```diff
@@ -0,0 +1,59 @@
+// Module included in the following assemblies:
+//
+// * observability/distr_tracing/distr_tracing_tempo/distr-tracing-tempo-configuring.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="distr-tracing-tempo-config-receiver-tls-for-tempomonolithic_{context}"]
+= Configuring the receiver TLS for a TempoMonolithic instance
+
+* You can provide a TLS certificate in a secret and configure it in the `TempoMonolithic` custom resource.
++
+[NOTE]
+====
+This feature is not supported with the enabled Tempo Gateway.
+====
++
+.TLS for receivers and using a user-provided certificate in a secret
+[source,yaml]
+----
+ apiVersion: tempo.grafana.com/v1alpha1
+ kind:  TempoMonolithic
+# ...
+    spec:
+# ...
+      ingestion:
+        otlp:
+          grpc:
+            tls:
+              enabled: true # <1>
+              certName: <tls_secret> # <2>
+              caName: <ca_name> # <3>
+# ...
+----
+<1> TLS enabled at the Tempo Distributor.
+<2> Secret containing a `tls.key` key and `tls.crt` certificate that you apply in advance.
+<3> Optional: CA in a config map to enable mutual TLS authentication (mTLS).
+
+* Alternatively, you can use OpenShift's service serving certificates.
```

Comment:
We don't normally say just "OpenShift". Should these instances be {product-title}?

Reply:
Thank you for the feedback.
I've rewritten it using the `{product-title}` attribute.